Src/LiquidProjections/EventMap.cs

```diff
@@ -73,6 +76,22 @@ internal void AddFilter(Func<object, TContext, Task<bool>> filter)
             }
         }
 
+        private List<Handler> GetHandlersForType(Type eventType)
+        {
+            if (mappings.TryGetValue(eventType, out var handlers))

```

Comment:
This implies that if there's an exact match for the more specific event, it won't even look at the more generic one. In the old VisionSuite denormalizers, we would invoke both handlers.

Reply:
Yeah, I agree, invoking both handlers would be better